kaggle_environments/envs/mab/mab.js

```diff
@@ -0,0 +1,100 @@
+async function renderer(context) {
```

Comment:
This renderer is a clone of RPS with minor changes. Even though the environment supports n agents, the renderer only supports 2 currently so we'd need updates here if we wanted to support n agents in production.

Reply:
If anyone's interested, it could be nice to add a little visualization here or something a little snappier than plain text.